pages/dashboard/index.js

```diff
@@ -34,6 +38,41 @@ module.exports = settings => {
     next();
   });
 
+  app.get('/', (req, res, next) => {
+    const adminEsts = req.user.profile.establishments.filter(est => {
+      if (est.role === 'admin' || userIsNtcoAtEst(req.user.profile, est.id)) {
+        return true;
+      }
```

Comment:
I realised pretty much as soon as I merged this, that it probably wants to be based on `pil.review`/`pil.update` permissions rather than a direct query.

Reply:
That should be fairly easy though as it'll be in the scope of `allowedActions`

Reply:
I'm not sure it will be - as the licence holder has pil.update and may not be ntco or admin

Reply:
The licence holder won't have `pil.update` in their `allowedActions` for the establishment though (because `allowedActions` only contains globally valid permissions), so that's fine.

Reply:
👍